pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/AbstractApexNode.java

```diff
@@ -51,13 +51,6 @@ protected void handleSourceCode(String source) {
         // default implementation does nothing
     }
 
-    @Deprecated
-    @InternalApi
-    @Override
-    public T getNode() {
```

Comment:
I agree that this as good as any for `getNode` to go from `@Deprecated` to gone. But really this call belongs to the PMD maintainers... who I hope will agree...

One consequence of removal is that any remaining uses (i.e. custom rules) will break when the JAR is loaded (with some JVM error) rather than break at runtime (with an exception, or whatever a hypothetical dummy implementation would do). SGTM.